src/translator/translator.js

```diff
@@ -11,6 +11,26 @@ class GoogleTranslator {
   constructor(options) {
     this.defaultLanguage = options?.defaultLanguage || config.defaultLanguage;
     this.apiKey = config.googleTranslationApiKey;
+    this._cachedSupportedLanguages = [];
+    this._fetcherInterval;
+    this._initialized = false;
+  }
+
+  async init(force) {
+    if (!force && this._initialized) {
+      throw new Error('This instance is already initialized!');
+    }
+
+    const fetcher = async () => {
+      this.getSupportedLanguages().then(result =>
+        console.log(`Fetched ${result.length} supported languages`)
```

Comment:
Could you replace this with `logger.info()` like in other files?

Reply:
oh right I forgot habla have its own logger